imageio/imageio-tiff/src/main/java/com/twelvemonkeys/imageio/plugins/tiff/CCITTFaxDecoderStream.java

```diff
@@ -150,15 +150,21 @@ public CCITTFaxDecoderStream(final InputStream stream, final int columns, final
     }
 
     static int findCompressionType(final int encodedType, final InputStream stream) throws IOException {
+        return findCompressionType(encodedType, stream, TIFFBaseline.FILL_LEFT_TO_RIGHT);
+    }
```

Comment:
I don't see the need for keeping the old method. It's just used in tests. You can inline it in the tests, for easy migration.